workflows.py

```diff
@@ -0,0 +1,74 @@
+# @@@SNIPSTART python-money-transfer-project-template-workflows
+from datetime import timedelta
+from temporalio import workflow
+from temporalio.common import RetryPolicy
+from temporalio.exceptions import ApplicationError
+
+from shared import PaymentDetails
+from banking_service import InvalidAccountError, InsufficientFundsError
+from activities import WithdrawFailedError, DepositFailedError, RefundFailedError
+
+# Import activity, passing it through the sandbox without reloading the module
+with workflow.unsafe.imports_passed_through():
+    from activities import withdraw, deposit, refund
+
+@workflow.defn
+class MoneyTransfer:
+    @workflow.run
+    async def run(self, payment_details: PaymentDetails) -> str:
+        retry_policy: RetryPolicy = RetryPolicy(
+            initial_interval=timedelta(seconds=1),
+            backoff_coefficient=2,
+            maximum_attempts=3,
+            maximum_interval=timedelta(seconds=2),
+            non_retryable_error_types=["InvalidAccountError", "InsufficientFundsError"]
+        )
```

Comment:
```suggestion
        retry_policy: RetryPolicy = RetryPolicy(
            maximum_attempts=3,
            maximum_interval=timedelta(seconds=2),
            non_retryable_error_types=["InvalidAccountError", "InsufficientFundsError"]
        )
```

Reply:
In the Go version we're showing all the options even if they're default. Wanted to mirror that here.

Reply:
I don't think we should have in Go either, but 🤷